week1/homework/src/index.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
 // Write the homework code in this file
+let http = require('http');
+let port = 8080;
+let state = 10;
+
+
+let server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'text/html')
+    switch (req.url) {
+        case '/' :
+        break;
+        case '/state':
+            res.write(`<h1> OK :${state}</h1>`);
+            break;
+        case '/add':
+            res.write(`<h1> oK :${state + 1}</h1>`);
```

Comment:
You are not modifying the `state`. If I call `/add` twice after starting the program I expect `state` to be 12. In your code `state` is always 10.